### PR TITLE
More updates to RBAC documentation

### PR DESCRIPTION
**Pull request by potocnyj**
Remove references to the Administrator role since it is being removed, 
and also mention that the Owners team membership can be modified.

### CODE REVIEW

**Comment by potocnyj** [comment]
@gkristic @mtrias I think these updates will make the documentation pretty much ready to go.

@Preetam maybe you can take a look too 😄

File: content/general-reference/environments-team.md [hunk 1]
```diff
@@ -51,9 +51,10 @@ doing so will allow you to grant new users access to an environment by default.
 
 ### The *Owners* Team
 
-The *Owners* team cannot be deleted, renamed, or updated. It is the team of people that have
-permissions to do anything in the account. Among other things this includes setting up billing,
-removing people from teams, and even shutting down the account.
+The *Owners* team cannot be deleted or renamed. The users affiliated with it can be updated though;
```
Comment (Preetam) [inline content/general-reference/environments-team.md]: Don't need `though`.

File: content/general-reference/environments-team.md [hunk 2]
```diff
@@ -51,9 +51,10 @@ doing so will allow you to grant new users access to an environment by default.
...
+doing so will grant those users access to all environments by default. For enterprise-level organizations,
+it is the team of people that have permissions to do anything in the account. Among other things
```
Comment (Preetam) [inline content/general-reference/environments-team.md]: Need a comma after `things`.

File: content/general-reference/environments-team.md [hunk 3]
```diff
@@ -62,4 +63,5 @@ go to the *Settings* page, to the *Teams* tab under the *Environment Settings* s
 
 [![Environment's Teams](/img/docs/rbac/env-teams.png)](/img/docs/rbac/env-teams.png)
 
-For more information about how permissions work refer to [Role-Based Access Control](/general-reference/rbac/)
+For enterprise-level organizations, Teams can be more assigned roles with different permissions.
```
Comment (Preetam) [inline content/general-reference/environments-team.md]: "Teams can be more assigned roles with different permissions" sounds weird.

File: content/general-reference/environments-team.md [hunk 4]
```diff
@@ -62,4 +63,5 @@ go to the *Settings* page, to the *Teams* tab under the *Environment Settings* s
...
+For information about how permissions work refer to [Role-Based Access Control](/general-reference/rbac/).
```
Comment (Preetam) [inline content/general-reference/environments-team.md]: Need a comma after `work`.

File: content/general-reference/environments-team.md [hunk 5]
```diff
@@ -51,9 +51,10 @@ doing so will allow you to grant new users access to an environment by default.
 
 ### The *Owners* Team
 
-The *Owners* team cannot be deleted, renamed, or updated. It is the team of people that have
-permissions to do anything in the account. Among other things this includes setting up billing,
-removing people from teams, and even shutting down the account.
+The *Owners* team cannot be deleted or renamed. The users affiliated with it can be updated;
```
Comment (gkristic) [inline content/general-reference/environments-team.md]: "The users affiliated with it can be updated" is ambiguous. It could refer to managing who's on the team, or to updating each user itself. I'd rephrase this as "The Owners team cannot be deleted or renamed, but users can be added to, or removed from it. Users in the Owners team are granted access to all environments by default."

File: content/general-reference/environments-team.md [hunk 6]
```diff
@@ -62,4 +63,5 @@ go to the *Settings* page, to the *Teams* tab under the *Environment Settings* s
 
 [![Environment's Teams](/img/docs/rbac/env-teams.png)](/img/docs/rbac/env-teams.png)
 
-For more information about how permissions work refer to [Role-Based Access Control](/general-reference/rbac/)
+For enterprise-level organizations, Teams can be assigned roles with different permissions.
```
Comment (gkristic) [inline content/general-reference/environments-team.md]: "Teams" should be all lowercase. It's a simple noun.

File: content/general-reference/rbac.md [hunk 7]
```diff
@@ -37,13 +37,14 @@ to the "Staging" environment, but only read access to the "Production" environme
...
+There are currently three roles available:
 
-- *Administrator*
+- *Owner*
 - *Read-Write*
 - *Read-Only*
 
-The *Administrator* role allows full access.
+The *Owner* role grants full access to an organization. This role is special, and cannot
+be assigned to teams other than [the Owners Team](/general-reference/environments-team/#the-owners-team).
```
Comment (gkristic) [inline content/general-reference/rbac.md]: "... to teams other than the Owners Team" sounds redundant to me, and the uppercase in "Team" is not needed. I'd put the Owners team as a first class citizen, not only qualifying the "team" noun, like "... be assigned to teams other than Owners".
